lib/cli/eventually-list-version.js

```diff
@@ -22,9 +24,39 @@ module.exports = async () => {
     if (!cliParams.has('-v')) return false;
   }
 
+  const localServerlessPath = await resolveLocalServerlessPath();
+
+  if (localServerlessPath) {
+    // If the version is already local, do not try to fallback for version resolution to avoid falling into the loop
+    const isLocal =
+      path.resolve(localServerlessPath, '../cli/eventually-list-version.js') === __filename;
+    if (!isLocal) {
+      // Attempt to resolve version with local Serverless instance
+      process.stdout.write(
+        `Serverless: ${chalk.yellow(
+          'Running "serverless" installed locally (in service node_modules)'
+        )}\n`
+      );
+      try {
+        try {
+          return require(path.resolve(localServerlessPath, '../../bin/serverless.js'));
+        } catch {
+          // Pass and attempt to use `bin/serverless` that was used by older version of the Framework
+          return require(path.resolve(localServerlessPath, '../../bin/serverless'));
+        }
+      } catch {
+        // This is just a fallback as for most (all?) versions it shouldn't happen
+        throw new ServerlessError(
+          'Could not resolve path to locally installed serverless.',
+          'INVALID_LOCAL_SERVERLESS_PATH'
+        );
```

Comment:
I'm starting to lose my sanity while trying to satisfy `eslint` that keeps throwing:
```
52:50  error  Expected to return a value at the end of async arrow function  consistent-return
```

I think I've tried like 10 different approaches here - do you see an obvious mistake here or maybe have a suggestion on how to satisfy `eslint`? I believe it's not possible for this function to not return (or throw an error).